pages/osx/stern.md

```diff
@@ -0,0 +1,24 @@
+# Stern
+
+> Tail Logs from multiple pods and multiple containers on Kubernetes.
+> More information: <https://github.com/wercker/stern>.
+
+- Tail logs from specific namespace:
+
+`stern <POD> --container <CONTAINER> --namespace <NAMESPACE>`
+
+- Tail logs using particular kubeconfig:
+
+`stern <POD> --container <CONTAINER> --kubeconfig <KUBECONFIG_PATH>`
+
+- Tail logs since last 2 hours:
```

Comment:
Perhaps this will read better?
```suggestion
- Display the last 2 hours worth of logs:
```

Reply:
i guess "Tail" is correct word because it displays last 2 hours worth of logs and tails for newer ones.
how about "Display the last 2 hours worth of logs and Tails for newer ones" ?

Reply:
Ah, I see @senani-mayank. What about this then?
```suggestion
- Display the last 2 hours worth of logs, tailing for newer ones:
```

Reply:
that seems fine @sbrl .
thanks for suggestion.